### DISCUSSION TITLE
Creating Synthetic MRI Data

### DISCUSSION THREAD

**Discussion by chaidosa** [dipy/dipy#2724] [#GSoC/GSoD]
Hi Everyone, I am Pritesh Verma (Deep Learning Researcher) I am interested in the project "Creating Synthetic MRI data"
I would like to get in touch with mentor of the same @skoudoro and Jong Sung Park.
Recently I was reading about the diffusion model in the literature. We have lot of research going around this field. I would love to contribute on this project and learn with this opportunity.

**Comment by pjsjongsung** [comment]
Hey @priteshvermaa ! Thank you for taking interest. We would love to have contributors. What kind of experience do you have on DL? (e.g. worked with medical/volumetric images, what kind of task, etc).

Also are you currently a student? We were meaning to propose this as a GSoC project, which could be beneficial for both DIPY and you if we can utilize GSoC.

**Reply by chaidosa** [reply]
Thank you @pjsjongsung for your response. 

In the field of DL, I have gained experience in a variety of areas, including image classification, diffusion modeling(Pytorch Implementation), Physics informed neural networks, neural ODE's, as well as linear algebra and optimization problems. I feel confident working in both research and development settings.

Currently, I am serving as a Post-baccalaureate fellow at RBCDSAI, which is one of India's leading AI research labs. Although my tenure at RBCDSAI is coming to an end, I am excited about the prospect of contributing to something meaningful this summer. Moreover, I am applying for GRAD School Next Spring (2024) and Fall (2024), and I believe that the opportunity to participate in this project through GSOC would be invaluable to my applications. If given the chance, I am confident that I can both learn from and make a significant contribution to this project.

**Reply by pjsjongsung** [reply]
Great! We would love to learn more about your skill set and what you can contribute with. Could you send an email to jp109@iu.edu with info on what time/date is best for you to talk about the project on zoom?

**Reply by chaidosa** [reply]
sure, I'll send the for the same.

**Reply by skoudoro** [reply]
Before talking about the project,  it would be great if you could start to contribute to the project also @priteshvermaa. 

There are many beginner friendly issues to fix. This is a requirement from PSF (Python Software Foundation). 

it is way to get familiar with the library. You can also check if you can generate the tests, etc. Let us know is you have any question during this process

**Reply by chaidosa** [reply]
Sure @skoudoro , I'll do the needful.
